fastlane_core/lib/fastlane_core/configuration/config_item.rb

```diff
@@ -1,6 +1,6 @@
 module FastlaneCore
   class ConfigItem
-    attr_accessor :key, :env_name, :description, :short_option, :default_value, :verify_block, :optional, :conflicting_options, :conflict_block, :deprecated, :sensitive
+    attr_accessor :key, :env_name, :description, :short_option, :default_value, :verify_block, :optional, :conflicting_options, :conflict_block, :deprecated, :sensitive, :allow_shell_conversion
```

Comment:
nvm, those are different, as some parameters in `initialize` are not exposed as `attr_accessor` and vice-versa, so we'll have to keep both anyway.

Done in 085ae69f2119ea3677bbffc3ceeb1ffcd3fce8b5